Assets/Prefabs/Robot.prefab

```diff
@@ -358,7 +358,7 @@ MonoBehaviour:
     creationDelayMs: 0
     laserDevice: /dev/tty_in1
     motorPort: outC
-    laserDutyCycle: -44
+    laserDutyCycle: 44
```

Comment:
ok @bmegli, at some point just implement the pid in xv11lidar library and change this to rpm setting